src/compiler/checker.ts

```diff
@@ -2121,12 +2121,16 @@ namespace ts {
             }
 
             function buildTypeDisplay(type: Type, writer: SymbolWriter, enclosingDeclaration?: Node, globalFlags?: TypeFormatFlags, symbolStack?: Symbol[]) {
-                const globalFlagsToPass = globalFlags & TypeFormatFlags.WriteOwnNameForAnyLike;
+                let globalFlagsToPass = globalFlags & TypeFormatFlags.WriteOwnNameForAnyLike;

```

Comment:
not at all. So this is just address an issue for release-2.0.5 the master uses accessibility check (check accessibility allows we to correctly check local type)